packages/react-charting/src/types/IDataPoint.ts

```diff
@@ -632,3 +632,8 @@ export interface ICustomizedCalloutData {
   x: number | string | Date;
   values: ICustomizedCalloutDataPoint[];
 }
+
+/** ILineChartPoints with required color prop */
+export type IModifiedLineChartPoints = ILineChartPoints & {
```

Comment:
why is IModifiedLineChartPoints needed? Why cant we use ILineChartPoints only.

Reply:
We can use ILineChartPoints only, but IModifiedLineChartPoints makes type-checking easy if the color is already set. Otherwise, we will have to use `color!` to bypass lint error.
Only AreaChart and LineChart receive `lineChartData` with type ILineChartPoints[], so we can remove the export and use IModifiedLineChartPoints directly in the components.
What should I do?